classes/Rijndael.php

```diff
@@ -48,24 +48,47 @@ public function encrypt($plaintext)
         if ($length >= 1048576) {
             return false;
         }
-        return base64_encode(mcrypt_encrypt(MCRYPT_RIJNDAEL_128, $this->_key, $plaintext, MCRYPT_MODE_CBC, $this->_iv)).sprintf('%06d', $length);
+        $ciphertext = null;
+        if (function_exists('openssl_encrypt') && version_compare(phpversion(), '5.3.3', '>=')) {
+            $ciphertext = openssl_encrypt($plaintext, 'AES-128-CBC', $this->_key, OPENSSL_RAW_DATA, $this->_iv);
+        } elseif (function_exists('mcrypt_encrypt') && version_compare(phpversion(), '5.3.3', '<')) {
```

Comment:
@jocel1 you're right I'll change it :+1: 
sorry for the late response :/ I didn't get the notification.